DependencyInjection/Configuration.php

```diff
@@ -23,6 +23,13 @@ public function getConfigTreeBuilder()
         $rootNode->setBuilder(new AllowExtraPropertiesNodeBuilder());
 
         $rootNode->children()
+            ->arrayNode('allowedParamConverters')
+                ->prototype('array')
+                    ->children()
+                        ->scalarNode('name')->isRequired()->end()
```

Comment:
Ok, although i don't fully understand why we have the `json_schema` level then :smirk: